File: Spartane.Web/Uploads/Scripts/Registro_de_Usuario.js
$(document).ready(function () {
    var inpuElementArray = [
        { "inputId": "Clave", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Nombres", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Apellido_Paterno", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Apellido_Materno", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Nombre_Completo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Usuario", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Contrasena", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Confirmar_Contrasena", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Correo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Celular", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Codigo_Verificacion_Correo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Codigo_Verificacion_Celular", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Codigo_Reestablecer_Contrasena", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "El_correo_fue_Validado", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }
        ,{ "inputId": "El_Celular_fue_Validado", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }

    ];
    setInputEntityAttributes(inpuElementArray, "#", 1);
});

File: Spartane.Core/Domain/Registro_de_Usuario/Registro_de_Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Web.Mvc;

namespace Spartane.Core.Domain.Registro_de_Usuario
{
    /// <summary>
    /// Registro_de_Usuario table
    /// </summary>
    public class Registro_de_Usuario: BaseEntity
    {
        public int Clave { get; set; }
        public string Nombres { get; set; }
        public string Apellido_Paterno { get; set; }
        public string Apellido_Materno { get; set; }
        public string Nombre_Completo { get; set; }
        public string Usuario { get; set; }
        public string Contrasena { get; set; }
        public string Confirmar_Contrasena { get; set; }
        public string Correo { get; set; }
        public string Celular { get; set; }
        public string Codigo_Verificacion_Correo { get; set; }
        public string Codigo_Verificacion_Celular { get; set; }
        public string Codigo_Reestablecer_Contrasena { get; set; }
        public bool? El_correo_fue_Validado { get; set; }
        public bool? El_Celular_fue_Validado { get; set; }


    }
	
	public class Registro_de_Usuario_Datos_Generales
    {
                public int Clave { get; set; }
        public string Nombres { get; set; }
        public string Apellido_Paterno { get; set; }
        public string Apellido_Materno { get; set; }
        public string Nombre_Completo { get; set; }

		
    }

	public class Registro_de_Usuario_Datos_para_Iniciar_Sesion
    {
                public int Clave { get; set; }
        public string Usuario { get; set; }
        public string Contrasena { get; set; }
        public string Confirmar_Contrasena { get; set; }

		
    }

	public class Registro_de_Usuario_Datos_para_validar_tu_cuenta
    {
                public int Clave { get; set; }
        public string Correo { get; set; }
        public string Celular { get; set; }

		
    }

	public class Registro_de_Usuario_Validacion
    {
                public int Clave { get; set; }
        public string Codigo_Verificacion_Correo { get; set; }
        public string Codigo_Verificacion_Celular { get; set; }
        public string Codigo_Reestablecer_Contrasena { get; set; }

		
    }

	public class Registro_de_Usuario_Logica
    {
                public int Clave { get; set; }
        public bool? El_correo_fue_Validado { get; set; }
        public bool? El_Celular_fue_Validado { get; set; }

		
    }


}

